python/mxnet/_numpy_op_doc.py

```diff
@@ -173,3 +173,112 @@ def _np_cumsum(a, axis=None, dtype=None, out=None):
         `axis` is not None or `a` is a 1-d array.
     """
     pass
+
+
+def _np_mean(a, axis=None, dtype=None, out=None, keepdims=None):
+    """
```

Comment:
Add the real signature here: `mean(a, axis=None, dtype=None, out=None, keepdims=None)` and an empty line after it.